doc/index.rst

```diff
@@ -1,40 +1,84 @@
-.. role:: html(raw)
-   :format: html
+PennyLane Documentation
+=======================
 
-:html:`<br>`
-
-
-.. image:: _static/pennylane_thin.png
-    :align: center
-    :width: 100%
-    :target: javascript:void(0);
-    :alt: PennyLane
-
------------------------------------------------
+.. rst-class:: lead grey-text ml-2
 
+:Release: |release|
 
 .. raw:: html
 
     <style>
-    h1 {
-        display: none;
-    }
+        #right-column.card {
+            box-shadow: none!important;
+        }
+        #right-column.card:hover {
+            box-shadow: none!important;
+        }
+        .breadcrumb {
+            display: none;
+        }
+        h1 {
+            text-align: center;
+        }
+        .footer-relations {
+            border-top: 0px;
+        }
     </style>
-
-
-PennyLane
-=========
-
-
-:Release: |release|
-:Date: |today|
-
-PennyLane is a cross-platform Python library for quantum machine learning,
-automatic differentiation, and optimization of hybrid quantum-classical computations.
-
-
-:html:`<h2>Features</h2>`
-
+    <div class="container mt-2 mb-2">
+        <p class="lead grey-text">
+            PennyLane is a cross-platform Python library for quantum machine learning,
+            automatic differentiation, and optimization of hybrid quantum-classical computations.
+        </p>
+        <div class="row mt-3">
+            <div class="col-lg-4 mb-2 adlign-items-stretch">
+                <a href="introduction/pennylane.html">
+                    <div class="card rounded-lg" style="height:100%;">
+                        <div class="d-flex">
+                            <div>
+                                <h3 class="card-title pl-3 mt-4">
+                                Using PennyLane
+                                </h3>
+                                <p class="mb-3 grey-text px-3">
+                                    Learn PennyLane through short code snippets <i class="fas fa-angle-double-right"></i>
```

Comment:
This doesn't really fit. Perhaps "A guided tour of the core features of PennyLane"?

Reply:
Changed